### PR TITLE
CONTRIBUTION: update signed-off-by

### PR DESCRIPTION
**Pull request by siddontang**
Signed-off-by: siddontang <siddontang@gmail.com>

## What have you changed? (mandatory)

Now TiKV enables DCO check, so we need to update the contribution doc for it.

## What are the type of the changes? (mandatory)

- Contribution

## How has this PR been tested? (mandatory)

No 

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

No

## Does this PR affect tidb-ansible update? (mandatory)

No

## Add a few positive/negative examples (optional)

PTAL @caniszczyk @ceohockey60 @queenypingcap 

BTW, do we need to add a git hook to do it automatically? 

E.g. adding following to the githook commit-msg

```
#!/bin/sh

NAME=$(git config user.name)
EMAIL=$(git config user.email)

if [ -z "$NAME" ]; then
    echo "empty git config user.name"
    exit 1
fi

if [ -z "$EMAIL" ]; then
    echo "empty git config user.email"
    exit 1
fi

git interpret-trailers --if-exists doNothing --trailer \
    "Signed-off-by: $NAME <$EMAIL>" \
    --in-place "$1"
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,3 +63,9 @@ If the change affects many subsystems, you can use ```*``` instead, like ```*:``
 For the why part, if no specific reason for the change,
 you can use one of some generic reasons like "Improve documentation.",
 "Improve performance.", "Improve robustness.", "Improve test coverage."
+
+### Signing off the Commit
+
+The project now enables [DCO check](https://github.com/probot/dco#how-it-works) and the commit message must contain `Signed-off-by` line for [Developer Certificate of Origin](https://developercertificate.org/)
```
Comment (queenypingcap) [inline CONTRIBUTING.md]: `Signed-off-by` line -> a `Signed-off-by` line

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -63,3 +63,9 @@ If the change affects many subsystems, you can use ```*``` instead, like ```*:``
...
+
+You can using option `-s` for `git commit` to automatically add a `Signed-off-by` to the commit message.
```
Comment (queenypingcap) [inline CONTRIBUTING.md]: You can using option `-s` -> You can use the `-s` option

**Comment by smallyard** [comment]
You use the -s option, but the check is still not pass.
#3421

**Comment by siddontang** [comment]
@smallyard 

you must sign off all the commits, not only one. 

Now the DCO is not required forcibly for the old PRs. We can merge yours even not signed.

**Comment by smallyard** [comment]
@siddontang  I get it. please merge that PR, so I needn't  post another.

**Comment by caniszczyk** [comment]
LGTM

**Comment by siddontang** [comment]
please help me approve it @caniszczyk

**Pull request merged**